src/tactic/doc_commands.lean

```diff
@@ -181,39 +183,32 @@ meta def tactic.get_tactic_doc_entries : tactic (list tactic_doc_entry) :=
 attribute.get_instances `tactic_doc >>=
   list.mmap (λ dcl, mk_const dcl >>= eval_expr tactic_doc_entry)
 
-/-- `add_tactic_doc tde` assumes `tde : pexpr` represents a term of type `tactic_doc_entry`.
-It adds a declaration to the environment with `tde` as its body and tags it with the `tactic_doc`
+/-- `add_tactic_doc tde` adds a declaration to the environment
+with `tde` as its body and tags it with the `tactic_doc`
 attribute. If `tde.decl_names` has exactly one entry, and the referenced declaration is missing a
 doc string, it adds `tde.description` as the doc string. -/
```

Comment:
```suggestion
attribute. If `tde.decl_names` has exactly one entry `` `decl`` and 
if `tde.description` is the empty string, `add_tactic_doc` uses the doc 
string of `decl` as the description. -/
```

Reply:
I didn't want to talk about `description` because it is an implementation detail and should IMHO not be used directly.

Reply:
Perhaps what I wrote could be improved, but unless I'm misunderstanding the code, I think the current doc string of this function is still incorrect.

Reply:
Oh, sorry, I missed this when I approved the PR. This is the doc string for the implementation, not the doc entry for the command. I think referencing `description` here is fine. And indeed, it's wrong as written.

Reply:
@gebner If you'd rather rephrase, feel free, before or after this gets merged.

Reply:
Sorry, I thought your review comment was for a different doc string, Bryan.  The change looks good.